TypeScript.py

```diff
@@ -814,6 +840,7 @@ def on_selection_modified(self, view):
                 self.setOnSelectionIdleTimer(1250)
             else:
                 self.setOnSelectionIdleTimer(50)
+            self.showTopLineStatus(info, view)
```

Comment:
Because changes of cursor positions and selections are not likely to introduce errors, it might be better to put the method call in ``on_modified`` instead of ``on_selection_modified``